Content/src/Server/ServerGiraffe.fs

```diff
@@ -34,12 +38,36 @@ let port =
     "SERVER_PORT"
 //#endif
     |> tryGetEnv |> Option.map uint16 |> Option.defaultValue 8085us
+#if (bridge)
+/// Elmish init function with a channel for sending client messages
+/// Returns a new state and commands
+let init clientDispatch () =
+    let value = {Value = 42}
```

Comment:
Formatting: let's keep a single space at beginning and end of  record construction `{ Value = 42 }` - this comment applies to other places as well